src/ui/styles/Overlay.module.css

```diff
@@ -0,0 +1,72 @@
+* {
```

Comment:
CSS Modules are supposed to contain "local" styles, so they should only declare class styles and no global styles. Global styles like this are what your `src/index.css` is for.